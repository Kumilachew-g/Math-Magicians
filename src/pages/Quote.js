import React from 'react';
// import ParticlesComponent from 'react-particles-js';

const Quote = () => (
  <div>
    <div className="quotes">
      <h3>
        Without mathematics, there’s nothing you can do. Everything around you
        is mathematics. Everything around you is numbers.
        <br />
        It is about understanding. — Shakuntala Devi.
      </h3>
    </div>
  </div>
);
export default Quote;
