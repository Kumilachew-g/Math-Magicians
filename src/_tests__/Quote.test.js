import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../pages/Quote';

it('expect quote to match working snapshot', () => {
  const quote = renderer.create(<Quote />).toJSON();
  expect(quote).toMatchSnapshot();
});
