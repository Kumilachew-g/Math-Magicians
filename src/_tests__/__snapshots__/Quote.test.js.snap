// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect quote to match working snapshot 1`] = `
<div>
  <div
    className="quotes"
  >
    <h3>
      Without mathematics, there’s nothing you can do. Everything around you is mathematics. Everything around you is numbers.
      <br />
      It is about understanding. — Shakuntala Devi.
    </h3>
  </div>
  <div
    id="tsparticles"
  >
    <canvas
      style={
        Object {
          "height": "100%",
          "width": "100%",
        }
      }
    />
  </div>
</div>
`;
